packages/react-native/React/Views/RCTFont.mm

```diff
@@ -407,7 +407,7 @@ + (UIFont *)updateFont:(UIFont *)font
     fontSize = font.pointSize ?: defaultFontSize;
     fontWeight = weightOfFont(font);
     isItalic = isItalicFont(font);
-    isCondensed = isCondensedFont(font);
+    isCondensed = isCondensedFont(font) || [familyName isEqualToString:@"SystemCondensed"];
```

Comment:
Can you move this inside `isCondensedFont` so it's consistent across all callers?

Reply:
it's not going to work unless we pass the familyName after parsing it with RCTConvert

we would need to modify the signature of the function for only 1 use case handled a few lines below

I think it's better to keep it separate, wdyt?